src/Graphics/OpenGLContext/GLSL/glsl_CombinerProgramBuilder.cpp

```diff
@@ -229,6 +229,7 @@ class VertexShaderHeader : public ShaderPart
 			std::stringstream ss;
 			ss << "#version " << Utils::to_string(_glinfo.majorVersion) << Utils::to_string(_glinfo.minorVersion) << "0 es " << std::endl;
 			ss << "# define IN in" << std::endl << "# define OUT out" << std::endl;
+			ss << "OUT highp float z;" << std::endl << "uniform lowp int uClampMode;" << std::endl;
```

Comment:
z is a bad name for global variable.
Please choose something else, OriginalZ for example.